webapi/src/main/scala/org/knora/webapi/responders/v1/ValuesResponderV1.scala

```diff
@@ -1399,11 +1466,20 @@ class ValuesResponderV1 extends ResponderV1 {
 
             response <- (storeManager ? SparqlSelectRequest(sparqlQuery)).mapTo[SparqlSelectResponse]
             rows: Seq[VariableResultsRow] = response.results.bindings
-        } yield sparqlQueryResults2LinkValueQueryResult(rows, userProfile)
+            maybeLinkValueQueryResult = sparqlQueryResults2LinkValueQueryResult(rows, userProfile)
```

Comment:
If I understand correctly, `sparqlQueryResults2LinkValueQueryResult` calls `PermissionUtilV1.getUserPermissionOnLinkValueV1WithValueProps` which always returns view permissions for standoff links (those are owned by the system user).

And since `PermissionUtilV1.getUserPermissionOnLinkValueV1WithValueProps` was not called before the changes you made here, no permissions could be granted to any user on a standoff link.